examples/standards/gbfs/Netex_gbfs_exm1.xml

```diff
@@ -174,8 +174,23 @@ GBFS file: vehicle_types.json:
 							<MaximumVelocity>14</MaximumVelocity>
 							<!-- NeTEX maximum velocity in m/s. -->
 							<VehicleCategory>scooter</VehicleCategory>
+							<NumberOfWheels>2</NumberOfWheels>
 							<!-- GBFS form_factor -->
 						</SimpleVehicleType>
+						<SimpleVehicleType version="any" id="cargo2">
+							<!-- GBFS vehicle_type_id -->
+							<Name>Cargo bike</Name>
+							<!-- GBFS name -->
+							<PropulsionType>electric</PropulsionType>
+							<!-- GBFS propulsion_type -->
+							<MaximumRange>12345</MaximumRange>
+							<!-- GBFS max_range_meters -->
+							<MaximumVelocity>30</MaximumVelocity>
+							<!-- NeTEX maximum velocity in m/s. -->
+							<VehicleCategory>cargoCycle</VehicleCategory>
+							<NumberOfWheels>2</NumberOfWheels>
+							<!-- GBFS form_factor -->
```

Comment:
Comment move up?

Reply:
will change it

Reply:
done

Reply:
What did you actually do, because the comment is still below any element.

Reply:
good question. perhaps I never committed the change...

Reply:
now better